public/stylesheets/site/2.0/15-group-comments.css

```diff
@@ -30,7 +30,6 @@ div.comment, li.comment {
 .comment div.icon {
   float: left;
   margin: -1.5625em 0.5625em 0.5625em 0;
-  border-bottom: 5px solid #ddd;
```

Comment:
Hmm. Is this intended to address the gap above the anonymous icon in Reversi? Because I'm pretty sure this change is going to remove the bottom border under all icons, which wouldn't be the correct way to fix that.

Reply:
I spoke with the tester, and you can just remove this change. They were referring to the border, not the gap, but the border itself is intentional. The color in Reversi may not be, but it's unrelated to this issue.